Resources/Prototypes/Catalog/Fills/Items/misc.yml

```
@@ -38,3 +38,12 @@
       item:
       - KukriKnife
 
+- type: entity
+  id: ClothingShoesBootsSyndieFilled
+  parent: ClothingShoesBootsCombat
+  suffix: Filled
```

```suggestion
  suffix: Filled, Throwing Knife
```
Also need similar suffix changes on default boots

---

> Also need similar suffix changes on default boots

Do you want me to do this to every entity in ``misc.yml`` ?
Or uuuh on ``ClothingShoesBootsSecFilled`` ?

---

No, only this type of boots